PurchasesTests/Attribution/AttributionFetcherTests.swift

```diff
@@ -161,7 +164,8 @@ class AttributionFetcherTests: XCTestCase {
     }
 
     func testPostAppleSearchAdsAttributionIfNeededSkipsIfATTFrameworkNotIncludedOnNewOS() {
-        if #available(iOS 14, macOS 11, tvOS 14, *) {
+        #if os(iOS)
```

Comment:
Since the `AdClient` class is only available for iOS, it doesn't make sense to launch these tests